python/katana/warnings_config.py

```diff
@@ -0,0 +1,13 @@
+import warnings
+
+
+def disable_partial_modin_warnings():
+
+    # Ignore any warnings related to defaulting to pandas for any operation.
```

Comment:
you had period at the end of this comment but not for other comments below :)

Reply:
X.X oops